docs/main/getting-started/installation.md

```diff
@@ -6,7 +6,7 @@ slug: /getting-started
 
 # Installing Capacitor
 
-You can create a new Capacitor application or add Capacitor to your existing web project. This can be done via CLI or using the [VS Code extension](vscode/getting-started).
+You can create a new Capacitor application or add Capacitor to your existing web project. This can be done via CLI or using the [VS Code extension](/main/getting-started/vscode-extension.mdx).
```

Comment:
Can we change this to "using the community-maintained [VS Code extension](...)" ?